smart-contracts/contracts/cl-vault/src/vault/range/move_position.rs

```diff
@@ -473,16 +473,24 @@ pub fn handle_iteration_create_position_reply(
         .target_range_position_ids
         .push(create_position_message.position_id);
 
+    let mrs = MODIFY_RANGE_STATE.load(deps.storage)?.unwrap();
```

Comment:
Should we also clean the MODIFY_RANGE_STATE, after its use is finished after merge.

Reply:
Yes, will add this

Reply:
changed in 8b2ffc162231d16d6d43e531728ab02ca4f69ced